### ISSUE TITLE
Analytics: Taxes Report Chart

### ISSUE THREAD

**Issue by timmyc** [woocommerce/wc-admin#923]
**Design Post**: p6riRB-3dT-p2
**Mock API**: [https://href.li/?https://app.swaggerhub.com/apis/peterfabian/wc-v3-api/1.0.0#/default/get_reports_taxes_stats](get_reports_taxes_stats)
**REST API Post**: p7bje6-15p-p2

This issue involves creation of the summary numbers and chart for the taxes report. Currently the REST API endpoints are not developed yet ( #844 ) Until then we should utilize the swagger hub endpoint.

**Comment by Aljullu** [comment]
@timmyc, is this issue the same that was implemented in #779?

**Comment by timmyc** [comment]
Yes you are correct @Aljullu 🤦‍♂️ i must have been running an old branch when i created this issue.
